### DISCUSSION TITLE
How to add custom message in http sender response

### DISCUSSION THREAD

**Discussion by partha1998** [nextgenhealthcare/connect#6386] [Q&A] [answered]
Hello i am using 
mirth server version : 4.5.2
operating system : linux
java version : 11

i have set up a channel http listener and http sender connectors , is there any way that i can set up some message and stauts code in http sender at transformer while hit some condition and stop the further execution like http sender send the request ?

**Comment by pacmano1** [comment]
Yes.   That is possible.  What have you tried?

**Reply by partha1998** [reply]
i have created a channel with http listener as source  and http sender as destination .At source transformation , i authenticate the token getting from http listener , what i want like if the token is invalid then at that time i sent the error status code with a error message to the client and want to block further processing of the message.

**Reply by pacmano1** [reply, marked as answer]
Your source tab indicates where and when to source a response.  Also, you can use variables for the response code.   Perhaps revisit https://docs.nextgen.com/bundle/Mirth_User_Guide_4_5_0/page/connect/connect/topics/c_HTTP_Listener_connect_ug.html.

For example, you can set a $r variable with the result you want for the response.  It will show in the response dropdown on the source tab.
